### ISSUE TITLE
Voice support in editor and input boxes

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#202763]
Around the workbench, it would be nice to have a command / key binding which allows activating a direct speech to text experience. This would particularly be useful in input boxes and markdown documents. 

This exists already for the terminal as `Terminal: Start Terminal Voice`.

cc @bpasero

**Comment by bpasero** [comment]
@meganrogge can you clarify how this issue is different from https://github.com/microsoft/vscode/issues/40976 ?

And besides, I would probably not overload the feature with editors + input fields because the implementation and experience will likely be very different. In places where an editor is used as input field (for example git commit box), I would think it could just work if we have it for editors.

**Comment by meganrogge** [comment]
@bpasero great point, they will be the same. 

I think the difference with the issues is this one requires an extension and I think that one is asking for native support?

**Comment by bpasero** [comment]
As far as I can tell, our story going forward is to require the speech extension but we may decide to make it easier for the user to discover it by e.g. showing the mic icon in Copilot chat and then triggering extension installation when needed.

In any case, I would not think that we should have 2 issues on this topic, even if the one asks for support without extension (which currently is unlikely given the size of the extension).
